README.md

```diff
@@ -1,19 +1,218 @@
+install and use on their GitHub organizations. However, Allstar can be run by
+anyone if need be, see [the operator docs](operator.md) for more details.
 
-## 
-**Scope**
+## **Quick start**
 
-Security related settings/files only. Allstar is not a general GitHub management bot.
+[Install Allstar](https://github.com/apps/allstar-app) on your organizations and
+repositories like any other GitHub App. Follow the [quick start
+instructions](quick-start.md) to setup the configuration files needed to enable
+Allstar on your repositories.
 
+These quick start instructions don't provide much explanation, for more details
+on all the configuration options, continue reading.
 
-# 
-**Get Involved**
+### [Help! I'm getting issues created by Allstar and I don't want them.](opt-out.md)
 
-A part of the [Securing Critical Projects WG](https://github.com/ossf/wg-securing-critical-projects).
+## **Enable Configuration**
+
+Allstar can be enabled on individual repositories at the app level, and
+additionally each security policy can be enabled or disabled as well. For
+organization-level configuration, create a repository named `.allstar` in your
+organization. Then create a file called `allstar.yaml` in that
+repository.
+
+Allstar can either be set to an opt-in or opt-out strategy. In opt-in, only
+those repositories explicitly listed are enabled. In opt-out, all repositories
+are enabled, and repositories may be explicitly listed to opt-out. Allstar is
+set to opt-in by default, and therefore is not enabled on any repository
+immediately after installation. To continue with the default opt-in strategy,
+list the repositories for Allstar to be enabled on in your organization like so:
+
+```
+optConfig:
+  optInRepos:
+  - repo-one
+  - repo-two
+```
+
+To switch to the opt-out strategy, set that option to true:
+
+```
+optConfig:
+  optOutStrategy: true
+```
+
+If you with to enable Allstar on all but a few repositories, you may use opt-out
+and list the repositories to disable:
+
+```
+optConfig:
+  optOutStrategy: true
+  optOutRepos:
+  - repo-one
+  - repo-two
+```
+
+### Repository Override
+
+Individual repositories can also opt in or out using configuration files inside
+those repositories. For example, if the organization is configured with the
+opt-out strategy, a repository may opt itself out by including the file
+`.allstar/allstar.yaml` with the contents:
+
+```
+optConfig:
+  optOut: true
+```
+
+Conversely, this allows repositories to opt-in and enable Allstar when the
+organization is configured with the opt-in strategy. Because opt-in is the
+default strategy, this is how Allstar works if the `.allstar` repository doesn't
+exist.
+
+In the organization-level `allstar.yaml`, repository override may be disabled with the setting:
+
+```
+optConfig:
+  disableRepoOverride: true
+```
+
+This allows an organization-owner to have a central point of approval for
+repositories to request an opt-out through a GitHub PR. Understandably, Allstar
```

Comment:
Not sure what "Allstar" is referring to? maybe `.allstar`?

Reply:
Referring to the app here. Just trying to say that some repos will need to opt-out of Allstar or individual policies, and using GitHub PRs on `.allstar` is a good way to have control over approval.